frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.jsx

```diff
@@ -126,11 +125,16 @@ export default class TagEditorParam extends Component {
     const hasSelectedDimensionField =
       isDimension && Array.isArray(tag.dimension);
     const hasWidgetOptions = widgetOptions && widgetOptions.length > 0;
+
     return (
-      <Card className="p2 mb2">
-        <h3 className="pb2 text-brand">{tag.name}</h3>
+      <div className="px3 pt3 mb1 rounded border-top">
+        <div className="mt1 mb2">
+          <h4 className="py1 px2 inline text-white bg-purple circular">
```

Comment:
I'm a 👎 on the purple background here. Ideally we could create hierarchy and make the tag name easier to spot without being quite so heavy handed. Might try using the text size or color instead of styling the container.